build.sh

```diff
@@ -29,8 +29,4 @@ for crate in "${CRATES[@]}"; do
 done
```

Comment:
It seems that I forgot to remove this file in #410. If you want you can delete it.

Reply:
You mean the whole script?

Reply:
Yes, it was replaced by the `justfile`.

Reply:
Thanks, I'll do that in a different PR.